treeherder/etl/management/commands/ingest.py

```diff
@@ -205,9 +209,10 @@ def query_data(repo_meta, commit):
     merge_base_commit = compareResponse.get("merge_base_commit")
     if merge_base_commit:
         commiter_date = merge_base_commit["commit"]["committer"]["date"]
-        # Since we don't use PushEvents that contain the "before" or "event.base.sha" fields [1]
-        # we need to discover the right parent which existed in the base branch.
-        # [1] https://github.com/taskcluster/taskcluster/blob/3dda0adf85619d18c5dcf255259f3e274d2be346/services/github/src/api.js#L55
+        # Since we don't use PushEvents that contain the "before" or "event.base.sha" fields [1] we need to discover
+        # the right parent which existed in the base branch. [1]
+        # https://github.com/taskcluster/taskcluster/blob/3dda0adf85619d18c5dcf255259f3e274d2be346/services/github
+        # /src/api.js#L55
```

Comment:
These changes in the comment should not be happening. Please put it the way it was.